### ISSUE TITLE
Apply an existing tag

### ISSUE THREAD

**Issue by gu4xin1m** [packetchaos/navi#31]
Hi.

It is possible to have the tag feature allow applying  an existing tag ?  

```
root@3a529608fa78:/usr/src/app# navi tag --c Randori --v high --file ips.csv

The object you tried to create may already exist
```

In this case both tag category and value already exist , I just need to apply it to a list of ips.

**Comment by packetchaos** [comment]
Yes.  I will add it to the backlog.

**Comment by packetchaos** [comment]
I updated the code to allow for updates.  However, know that tag rules are not updated by navi.  There are too many nuances to updating a tag rule.  

Instead the tag --file functionality looks for IPs in the navi.db and looks up the UUID to tag by UUID.  This will apply the tag to all of the IPs found in the navi.db.

The current functionality was moved to tagrule --file to align with the future of tagrule methods

Any updates on Tags that are dynamic will fail.
